packages/astro/src/vite-plugin-astro/index.ts

```diff
@@ -199,6 +199,18 @@ export default function astro({ settings, logger }: AstroPluginOptions): vite.Pl
 			const parsedId = parseAstroRequest(id);
 			// ignore astro file sub-requests, e.g. Foo.astro?astro&type=script&index=0&lang.ts
 			if (!parsedId.filename.endsWith('.astro') || parsedId.query.astro) {
+				// Special edge case handling for Vite 6 beta, the style dependencies need to be registered here take affect
```

Comment:
I am confused by the comment. 

Does that mean that this code will be removed once vite fixes it? If so, can we add a link to a possible issue and a `TODO`?

Reply:
Yes, it'll be removed when Vite fixes it. I'll add a link here